packages/components/src/components/modal/__tests__/modal.test.js

```diff
@@ -0,0 +1,21 @@
+import React             from 'react';
+import { render }        from '@testing-library/react';
+import                   '@testing-library/jest-dom/extend-expect'
+import { CSSTransition } from 'react-transition-group';
+import Modal             from '../modal.jsx';
+
+describe('Modal', () => {
+    it('renders without crashing' , () => {
+        render(<Modal/>)
+    })
+    // it('<Modal /> should have CSSTransition', () => {
```

Comment:
we are not sure how we are goin to implement the testing yet so i commented out here.